### PR TITLE
Add APM to Node servers

### PR DESCRIPTION
**Pull request by jamieparkinson**
Not sure why I didn't bother doing this before!

### CODE REVIEW

File: catalogue/webapp/app.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
+// This needs to be the first module loaded in the application
+if (process.env.NODE_ENV !== 'test') {
+  require('elastic-apm-node').start({
+    serviceName: process.env.NEXT_PUBLIC_APM_SERVICE_NAME,
+    ...require('@weco/common/services/apm/apmConfig'),
+  });
+}
+
```
Comment (jamesgorrie) [inline catalogue/webapp/app.js:8]: Just a thought, how might we do this if we remove this entrypoint and use standard nextjs?

**Review by jamesgorrie — DISMISSED** [review]
Do we want it in the identity app? I'd think so.

File: catalogue/webapp/app.js (hunk 1 above)
Comment (jamieparkinson) [reply catalogue/webapp/app.js:8]: I don't think we'd be able to do that tbh - think we'd need the custom server for this alone

**Review by jamesgorrie — APPROVED** [review]

File: catalogue/webapp/app.js (hunk 1 above)
Comment (jamesgorrie) [reply catalogue/webapp/app.js:8]: Why I am asking:

Screenshot 2021-09-24 at 11 44 42

-- [from here](https://nextjs.org/docs/advanced-features/custom-server)

**Pull request merged**